book/plugins/general-abilities/video-content-download/ios/video-content-download-ios.md

```diff
@@ -0,0 +1,156 @@
+## Video Content Downloads
+### Availible from SDK: 11.2.0
+
+1. <a href="#description">Description</a>
+2. <a href="#general">General</a>
+3. <a href="#implementation">Download button protocol implementation</a>
+4. <a href="#download-button-states-images3
+">Download button states images</a>
+* * *
+
+<a name="description" />
+
+##### Description
+`Video Content download` ability uses the provided atom entry video url to download and save the content locally for offline playback.
+***
+
+<a name="general" />
+
+##### General
+
+Downloading can be performed for 2 types of videos `m3u8`, `mp4`
+- being made by implementing the downloading mechanism introduced in iOS 10 (and improved in iOS 11)
+- downloaded item (including its model, images, video) stored on device and can be presented on dedicated screen, played locally, or removed.
+***
+
+<a name="protocol-implementation" />
+
+##### Pre-condition
+Downloads plugin should be added to the app.
+
+##### Protocol implementation
+1. Implement `ZPDownloadButtonDelegate` to prepare the download button appearance on your screen.
+
+	- `func downloadButton(_ downloadButton: ZPDownloadButtonProtocol, stateChanged state: ZPDownloadButtonState)`
+		- fired on download button state change, usually no action is required on this function.
```

Comment:
let's use `called` for the terminology. Also - please phrase it (and capitalize it) as a full sentence
Also - please add each function in a segment with 3 ` in order to keep it uniform.
Look at the login / analytics / player guide for expected structure.

Reply:
This goes for the rest of the functions